mvc-api/mvc-service/app/managers/CSVContentProcessor.java

```diff
@@ -0,0 +1,98 @@
+package managers;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.sunbird.search.util.SearchConstants;
+
+import java.io.*;
+import java.util.Arrays;
+
+public class CSVContentProcessor {
```

Comment:
Please use any CSV parsing opensource library to read and parse the data.
Also, I don't see any limit on # of lines in the file.

Reply:
But i have used BufferedReader which is used to process data with large csv files.
should limit on number of rows be implemented?